README.md

```diff
@@ -5,7 +5,65 @@ This repository contains the code to generate various versions of the Datadog Ag
 * https://s3.amazonaws.com/dd-agent/scripts/install_script_agent6.sh to install Agent 6
 * https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh to install Agent 7
 
-Usage instructions for the install script are in the [Datadog App](https://app.datadoghq.com/account/settings#agent/overview).
+
+## Agent install script usage
+Basic usage instructions for the Agent install script are in the [Datadog App](https://app.datadoghq.com/account/settings#agent/overview).
+
+### Agent configuration options
+The install script allows installation of different flavors of the Agent binaries and allows you to set some configuration options automatically. We list them below with their different possible values. 
+
+> [!IMPORTANT]
+> All variables are optional except `DD_API_KEY`, which is required unless `DD_UPGRADE` (upgrade from datadog-agent 5) is set.
+
+> [!WARNING]
+> The install script input options are only considered at initial installation and they don't overwrite pre-existing configuration files
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ 
```suggestion
> The install script input options are only considered at initial installation and they don't overwrite pre-existing configuration files.
```